src/app/content/hooks/receiveContent.ts

```diff
@@ -31,7 +35,11 @@ const hookBody: ActionHookBody<typeof receivePage> = (services) => async() => {
     return;
   }
 
-  const title = createTitle(page, book, intl);
+  if (!intlObj) {
+    intlObj = await createIntl(locale);
+  }
```

Comment:
@TomWoodward this is what I mentioned re `services.intl.current` not being defined yet when this hook runs